xdem/volume.py

```diff
@@ -354,7 +363,8 @@ def hypsometric_interpolation(voided_ddem: Union[np.ndarray, np.ma.masked_array]
 def local_hypsometric_interpolation(voided_ddem: Union[np.ndarray, np.ma.masked_array],
                                     ref_dem: Union[np.ndarray, np.ma.masked_array],
                                     mask: np.ndarray, min_coverage: float = 0.2,
-                                    count_threshold: Optional[int] = 1, plot: bool = False) -> np.ma.masked_array:
+                                    count_threshold: Optional[int] = 1, nodata: float = -9999,
```

Comment:
Pet peeve for linters: -9999 is an int ;) -9999. or -9999.0 is better!

Reply:
Yes, -9999 is an int, and I want it to be so, but nodata does not have to be an it. So I guess what I should do is add int as a possible type...